marketplace/cloudinary-assets/src/components/cloudinaryAppConfig/parameters.ts

```diff
@@ -0,0 +1,54 @@
+import get from 'lodash.get';
+
+export const MAX_FILES_UPPER_LIMIT = 25;
+const MAX_FILES_DEFAULT = 10;
+
+export interface InputParameters {
+  cloudName: string;
+  apiKey: string;
+  maxFiles: string;
+}
+
+export interface ExtensionParameters {
+  readonly cloudName: string;
+  readonly apiKey: string;
+  readonly maxFiles: number;
+}
+
+export type ParameterValue = string | boolean | number;
+
+export function toInputParameters(
+  parameters: Record<string, ParameterValue> | null
+): InputParameters {
+  return {
+    cloudName: `${get(parameters, ['cloudName'], '')}`,
+    apiKey: `${get(parameters, ['apiKey'], '')}`,
+    maxFiles: `${get(parameters, ['maxFiles'], MAX_FILES_DEFAULT)}`
+  };
+}
+
+export function validateParameters(parameters: InputParameters): string | null {
+  if (parameters.cloudName.length < 1) {
+    return 'Provide your Cloudinary Cloud name.';
```

Comment:
I noticed we used this kind of language in other places.
```suggestion
    return 'Provide your Cloudinary cloud_name.';
```